src/test/xMETRIC.t.sol

```diff
@@ -0,0 +1,147 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "forge-std/Test.sol";
+import "forge-std/Vm.sol";
+import "../contracts/xMETRIC.sol";
+
+/// @notice Throughout the contract we assume that Bob is the owner, Alice is any user
+contract xMetricTest is Test {
+    address alice = address(0xa);
+    address bob = address(0xb);
+
+    xMETRIC metricToken;
+
+    function setUp() public {
+        vm.label(alice, "Alice");
+        vm.label(bob, "Bob");
+
+        vm.startPrank(bob);
+        metricToken = new xMETRIC(1000000000 * 10**18);
+        vm.stopPrank();
+    }
+
+    function testExample() public {
+        assertTrue(true);
+    }
+
+    function test_GetTokenName() public {
+        console.log("Test ERC20 initialization");
```

Comment:
Does logging work in foundry?  do we need it - it's a test so I don't care regardless, normally this would go in a describe or it statement when testing with javascript but not sure how testing works here